### DISCUSSION TITLE
October Surprise

### DISCUSSION THREAD

**Discussion by Telefunken28** [iptv-org/community#1150] [Show and tell]
#EXTINF:0, group-title="CBS",CBS News Live
https://dai.google.com/linear/hls/event/-meo8FE1Q9ejM_TS2XK5ww/master.m3u8
#EXTINF:0, group-title="CBS",CBS News Live 2
https://cbsn-3-fastly.cbsnstream.cbsnews.com/out/v1/f081461c65494eeaa9ec596d0fe96a14/master.m3u8

#EXTINF:0, group-title="CSPAN",C-SPAN.org 1 standard-resolution
https://dotorg1.c-spanvideo.org/out/v1/87a3832998f64638a8f0db091cc1b8db/index.m3u8
#EXTINF:0, group-title="CSPAN",C-SPAN.org 2 standard-resolution
https://dotorg2.c-spanvideo.org/out/v1/49288f0956ae4b2ab871613ddff12962/index.m3u8
#EXTINF:0, group-title="CSPAN",C-SPAN.org 3 standard-resolution
https://dotorg3.c-spanvideo.org/out/v1/17e82e74517148d3911c24c973bbe0c8/index.m3u8
#EXTINF:0, group-title="CSPAN",C-SPAN.org 4 standard-resolution
https://dotorg4.c-spanvideo.org/out/v1/ce7d3bdab7ef40eaa72ca015e043f32e/index.m3u8
#EXTINF:0, group-title="CSPAN",C-SPAN.org 5 standard-resolution
https://dotorg5.c-spanvideo.org/out/v1/7080d4a4732245f6a43066125ba80ece/index.m3u8

#EXTINF:0, group-title="NATN",NATN1 [Sinclair National News Feed 1]
https://d23vujg5lkloml.cloudfront.net/out/v1/babd9358c71a4d65ae5987505e02eb36/index.m3u8
#EXTINF:0, group-title="NATN",NATN2 [Sinclair National News Feed 2]
https://d2xnglmjqh5c4.cloudfront.net/out/v1/98a94f2a721b490bb654f64117ae7ccc/index.m3u8

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#how-to-add-a-new-stream-link-to-a-playlists
